docs/users-guide/13-sql-parameters.md

```diff
@@ -161,8 +138,165 @@ WHERE True
   [[AND {% raw %}{{category}}{% endraw %}]]
 ```
 
+## The Field Filter variable type
+
+Setting a variable to the "Field Filter" type allows you to map the variable to a field in any table in the current database, and lets you create a "smart" filter widget that makes sense for that field.
+
+Field filter variables should be used inside of a `WHERE` clause in SQL, or a `$match` clause in MongoDB.
+
+### Field filter compatible types
+
+Field filters ONLY work with the following field types:
+
+- Category
+- City
+- Entity Key
+- Entity Name
+- Foreign Key
+- State
+- UNIX Timestamp (Seconds)
+- UNIX Timestamp (Milliseconds)
+- ZIP or Postal Code
+
+The field can also be a datetime, which can be left as "No semantic type" in the data model.
+
+When you set the __Variable type__ to "Field Filter", Metabase will present an option to set the __Field to map to__, as well as the __Filter widget type__. The options available for the Filter widget type depend on the field's type. For example, if you map to a field of type Category, you'll see options for either "Category" or None. If you map to a Date Field, you'll see options for None, Month and year, Quarter and year, Single date, Date range, or Date filter.
+
+If you don't want a widget on the question at all, which you might do, for example, if you just want to allow this question to be mapped to a dashboard filter (see more on that below), you can set the __Filter widget type__ to "None".
+
```

Comment:
Removing this, since we are likely going to remove this "functionality", since it's just insanely confusing.
```suggestion
```